src/index.html

```diff
@@ -1,13 +1,82 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="body">
+    <div class="container" data-qa="card">
+      <div class="container__image">
+
+        <img
+        class="container__image image"
+        src="./images/imac.jpeg"
+        alt="Imac"
+      >
+      </div>
+
+      <div class="container__info">
+        <h2 class="container__heading">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="container__text-small">
+          Product code: 195434
+        </p>
+
+        <div class="stars">
+          <div class="stars__4">
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="stars__reviews">
+            Reviews: 5
+          </div>
+        </div>
+
+        <div class="container__price">
+
+          <div class="container__price-text">
+            Price:
+          </div>
+
+          <div class="container__price-cost">
+            $2,199
+          </div>
+        </div>
+        <div>
+
+          <a
+            href="#"
+            class="container__buy"
+            data-qa="hover"
+          >
+          BUY
```

Comment:
don't write text in uppercase, make it uppercase using css
```suggestion
            BUY
```